### PR TITLE
Learning Data Science

### PR DESCRIPTION
**Pull request by ilanman**
Feel free to ignore. I want to review it once before you guys should read it.

### CODE REVIEW

**Comment by ilanman** [comment]
My main objective was to write something that is actionable for someone looking to break into DS/DA but also based on a framework.

Points I wanted to get across were:
* Don't over think the learning process, just do it and it'll come
* Learn from multiple sources
* Linear regression is a good place to start learning Stats that is useful and foundational for more complex stuff. Ditto for Logistic regression.
* Learn some basic analytics for business
* Python is good and DS folks don't code the same way as SE, and it's useful to know how SE's do it since you'll be working with them
* Think outside the traditional technical toolkit, e.g., I think behavior sciences is super cool and useful as a decision making framework. Which is in part what Data Science is used for.

Let me know if you think I hit these points and if it flows alright.

Also happy to shorten it. I wanted to add another paragraph about AB testing applied to Web tech business and AB testing applied to clinical trials (two different sets of problems, same framework) but figured the post was getting long.

File: content/post/ds_advice.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+A similar example exists when learning logistic regression. The statisticians have their approach to formulating this problem which involves assumptions on residuals and objects called logit and GLMs. Computer scientists use words like cross-entropy loss and sigmoid. Both approaches get you to the same set of weights which minimize the same error function, but one approach allows you to calculate odds-ratios and confidence intervals around your coefficients and the other is a basic building block of learning about neural networks. Learn about how other fields formulate the same problem, in part to know how it works, and in part to be able to read the literature as you learn more. Data Science is built upon various fields each with their own problem formulation and terminology. Don't get confused.
+
+This stuff all builds on itself - don’t take short cuts. Understand what you’re doing and why. Learn the basics, but learn them well.
```
Comment (mikekaminsky) [inline content/post/ds_advice.md]: >Understand what you’re doing and why.

I think we could provide more concrete tips on how to do this. What I normally say is that the best way to test understanding is to make predictions: "I think that in this model, if I change x, y will happen". Then I do it! If y happens, great, I understand what's going on, but _often_ y doesn't happen and I have to dig in and understand why. Doing this repeatedly is my favorite way to learn a new subject (which works especially well in programming domains where you can easily test your ideas with code!)

File: content/post/ds_advice.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+<h3>Build some business intuition</h3>
+
+Learn basic analytics. This includes cohort modeling, attribution and funnel analysis and how an A/B testing works. Know what a marketing function does and why they (should) work closely with the Analytics team. Why should Product managers care about sample sizes? It's not rocket science so know this stuff because you'll be expected to. I won't harp on this too much because most of it will come on the job. Don't rush it, but keep it in mind when you're building a dashboard or querying a database at the office. It's harder to learn this stuff on your own compared to Python basics or how Decision Trees work.
```
Comment (mikekaminsky) [inline content/post/ds_advice.md]: You could throw in some advice to learn some basic business fundamentals like how to read an income statement. I know that very few people do that, but for me personally that's been really helpful in linking my own work to how the business actually makes money.

File: content/post/ds_advice.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ 
+<h3>Keep learning</h3>
+
+Don't optimize locally. Get many perspectives (including on my advice!). Talk to many people in the industry. Go to Meetups, networking events, etc... Learn the vocabulary and start using it. Don't worry about being as efficient as possible in every endeavor all the time - you'll be working and learning for many years to come. Of course, it’s helpful to learn when you’ve wasted your time. It’ll happen. But learn from it. Don’t let the perfect be the enemy of the good. When you feel yourself veering away from the high density area of the Venn diagram, check if you're confident in your knowledge of the overlap and take small, incremental steps outwards.
+
+[^1]: Or emacs heaven help you.
```
Comment (mikekaminsky) [inline content/post/ds_advice.md]: lol

File: content/post/ds_advice.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+title: "Learn the overlaps: Advice for the Aspiring Data Scientist"
+author: "Ilan"
+cover: "/img/cover.jpg"
+tags: ["datascience"]
+date: 2018-10-01T21:29:29-04:00
+draft: true
+---
+
+I often get asked by junior data professionals who they can either break into data science or elevate their game. I try to give advice that is both generic enough that anyone can benefit and specific enough that is actionable for that person. Today I'll outline a generic framework for thinking about learning and provide a few concrete examples that support this way of thinking. These are tools that I still employ in my day to day learning and growing as a data professional.
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/ds_advice.md]: Thanks. `who` -> `how`

File: content/post/ds_advice.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+ 
+<!--more-->
+
+The learning framework that I trade in is echoed by Tyler Cowen’s sentiment in his podcast on the Ezra Klein show, when he recommend reading [piles of books](https://www.vox.com/2018/5/21/17369920/ezra-klein-show-book-recommendations-tyler-cowen-shakespeare-amazon) instead of any individual book on a given subject. Paraphrasing, he was getting at the idea that there isn’t really a single definitive, true source on anything. There are many sources of knowledge that overlap in various areas. The best bang for your buck exists in mastering that overlapping space. That is, you should strive to find the space of maximum overlap in the Venn diagram of the specific topic and learn that first. Once you feel comfortable with the content in the middle, begin to move outwards to the edges. In other words, the density of the overlap is roughly proportional to the value of the knowledge that lives there. 
+
+With that as a frame, in this post I will provide some pieces of advice on what every Data Scientist should be familiar with:
+
+* Python / R / coding skills
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/ds_advice.md]: I forgot but wanted to add that SQL is table stakes, like knowing Excel. I can add that as a sentence or footnote (I think footnote).

File: content/post/ds_advice.md (hunk 1 above)
Comment (ilanman) [reply content/post/ds_advice.md]: Agreed I can be more clear on what I mean. Following the two examples of LR and Logistic Reg, I was more getting at understanding underlying assumptions in the models one uses. What assumptions are made, and why? What happens when assumptions are violated? What does one approach get you that the other doesn't? But what you suggest does get at the notion of understanding it empirically.

File: content/post/ds_advice.md (hunk 2 above)
Comment (ilanman) [reply content/post/ds_advice.md]: Good point. We're having a lot of conversations here in trying to understand our business levers and their financial impact.

File: content/post/ds_advice.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
...piring DS's will likely download a [book](https://www.amazon.com/Python-Data-Analysis-Wrangling-IPython/dp/1449319793) about Data Analysis in Python or perhaps take [online](https://www.udemy.com/learning-python-for-data-analysis-and-visualization/?utm_source=adwords-learn&utm_medium=udemyads&utm_campaign=NEW-AW-PROS-TECH-US-DSA-1-EN-ENG_._ci__._sl_ENG_._vi_TECH_._sd_All_._la_EN_._&utm_content=deal4584&utm_term=_._ag_60674438611_._ad_267827191853_._de_c_._dm__._pl__._ti_dsa-304639795903_._li_9060351_._pd__._&gclid=EAIaIQobChMIjs2I3Yvo3QIVCIezCh0IQAd0EAAYAiAAEgIlf_D_BwE) [courses](https://www.coursera.org/learn/data-analysis-with-python). Luckily there is no shortage of good content. Don't stress much over which is the best one - remember to think in terms of piles not individual sources of knowledge. Pick one and work through the first few chapters which tend to cover basic concepts. Then download some datasets and hack on them using what you learned. Next, choose another book or course and repeat. Do this a few times and you'll notice where the overlap is. Then you can start getting into the later chapters where content tends to diverge. Just make sure you've locked in the basics.
```
Comment (mikekaminsky) [inline content/post/ds_advice.md]: Can we strip the UTM ugliness out of the Udemy link?

**Pull request merged**